nahusenayElias/Team_4
# DRD-44: Test Tailwind CSS

## Related ticket / Customer approval:
[DRD-44](https://edu-team4-react24k.atlassian.net/browse/DRD-44?atlOrigin=eyJpIjoiZDdjM2M0OTU0ZjIyNGIzNjk1OTQwYmI3N2NiZGZhNGMiLCJwIjoiaiJ9)

## In this PR:
- Installed Tailwind, postcss and autoprefixer as dev dependencies
- Installed @tailwindcss/typography to style text content coming from Drupal: "a utility-first plugin for Tailwind CSS that provides a set of prose classes to add beautiful typographic defaults to any vanilla HTML content."
- Imported the DM Sans font and defined it as default "font-sans" in tailwind.config.js
- This is simply testing Tailwind without implementing anything from the figma prototype, except for the font
## Testing instructions:
- cd to frontend
- npm install
- stop vite server (Ctrl + C in vite terminal)
- npm run dev
- Check styles are applied on frontend without errors

![Screenshot 2024-10-30 at 13 43 53](https://github.com/user-attachments/assets/7f11c8b3-384e-45c3-9ded-3589fd32c6fc)


[DRD-44]: https://edu-team4-react24k.atlassian.net/browse/DRD-44?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
